src/AaveV3ListGhoStewards_20230507/AaveV3ListGhoStewards_20230507_Payload.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+import {AaveV3Ethereum, IACLManager} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {GhoSteward} from 'gho-core/contracts/misc/GhoSteward.sol';
+import {IGhoToken} from 'gho-core/contracts/gho/interfaces/IGhoToken.sol';
+
+library Create2Helpers {
+  bytes32 public constant CREATE2_SALT = bytes32(0);
+
+  function deployGhoSteward(
+    address poolAddressProvider,
+    address ghoToken,
+    address riskCouncil,
+    address shortExecutor
+  ) external returns (address) {
+    GhoSteward _ghoStewardContract = new GhoSteward{salt: CREATE2_SALT}(
+      poolAddressProvider,
+      ghoToken,
+      riskCouncil,
+      shortExecutor
+    );
+    return address(_ghoStewardContract);
+  }
+
+  function getAddress(bytes memory bytecode) public pure returns (address) {
+    bytes32 hash = keccak256(
+      abi.encodePacked(
+        bytes1(0xff),
+        AaveGovernanceV2.SHORT_EXECUTOR,
+        CREATE2_SALT,
+        keccak256(bytecode)
+      )
+    );
+    return address(uint160(uint256(hash)));
+  }
+
+  function getBytecode(
+    address poolAddressProvider,
+    address ghoToken,
+    address riskCouncil,
+    address shortExecutor
+  ) public pure returns (bytes memory) {
+    bytes memory bytecode = type(GhoSteward).creationCode;
+    return
+      abi.encodePacked(
+        bytecode,
+        abi.encode(poolAddressProvider, ghoToken, riskCouncil, shortExecutor)
+      );
+  }
+}
+
+/**
+ * @title Gho Stewards AIP
+ * @author @aave - Aave Co
+ * @dev This proposal deploys GhoSteward, which consists of a set of actions.
+ * - Deployment of GhoSteward and transfers ownership to Aave DAO (short executor)
+ * - Grants role of PoolAdmin to GhoSteward
+ * - Grants role of BucketManager to GhoSteward
+ * Relevant governance links:
+ * - Governance: https://governance.aave.com/t/arfc-gho-steward-agile-parameter-changes/13922
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x67fa551993a94b801018d02154f3d4f27e29bea51fe6e862686de6bf9ee650af
+ */
+
+contract AaveV3ListGhoStewardsPayload is IProposalGenericExecutor {
```

Comment:
Let's change the name to `GhoAaveV3GhoSteward`.
All proposals about Gho can start with the preffix `Gho`